src/layouts/Default.vue

```diff
@@ -70,7 +72,10 @@ export default {
   },
 
   computed: {
-    ...mapState(['offline'])
+    ...mapState(['offline']),
+    ...mapState({
+      jobTheme: state => `job_theme--${state.app.jobTheme}`
```

Comment:
This `computed` property `jobTheme` contains the app job theme @oliver-sanders . Doesn't need to be necessarily here, but just to show where I had set it (though I just realized I used some cryptic syntax, not sure why... quite sure I can write just `mapState('app', ['jobTheme'])` instead; was probably running low on :coffee: )